aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -145,5 +169,12 @@ void OperatorEntry::updateCatchallDispatchTable_() {
   }
 }
 
+void OperatorEntry::updateVariableWrapper_() {
+  // precondition: kernelsMutex_ is locked
+  dispatchTable_.write([&] (DispatchTable& dispatchTable) {
+    dispatchTable.registerVariableWrapper(wrappers_.front());
```

Comment:
This probably crashes when there is no variable wrapper anymore because this got called after the last `deregisterVariableWrapper_`.